From 4db4159bbc949773be2c7ba28f055dbf513e5828 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Sun, 20 Oct 2024 23:38:23 -0700
Subject: [PATCH] Our CMAKE_INSTALL_LIBDIR already has prefix in it

---
 blosc2.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/blosc2.pc.in b/blosc2.pc.in
index d1e0626..66d5f5a 100644
--- a/blosc2.pc.in
+++ b/blosc2.pc.in
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+libdir=@CMAKE_INSTALL_LIBDIR@
 sharedlibdir=${libdir}
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
-- 
2.46.2

